services/avve_asset_apy/aave_apy.py

```diff
@@ -0,0 +1,34 @@
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+
+from utilities import cruize_constants
+
+
+class AaveApy:
+    def get_token_apys(self):
```

Comment:
Hey it seems we have both "token" & "asset" floating around our entire codebase. Let's stick to just one of them.
Either "token" or "asset", not both.

Reply:
are you saying about this one ??